backend/views/error.ejs

```diff
@@ -0,0 +1,3 @@
+<h1><%= message %></h1>
+<h2><%= error.status %></h2>
```

Comment:
Once my pr us merged I think you will need to pull in the latest main branch and potentially fix heaps of merge conflicts.

This pr is based on a version that is too old, lots of changes since then